CHANGES.txt

```diff
@@ -9,6 +9,12 @@ NOTE: The 4.2.0 Release of SCons will deprecate Python 3.5 Support. Python 3.5 s
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Jacob Cassagnol:
+    - Default hash algorithm check updated for Scons FIPS compliance. Now checks for hash viability
```

Comment:
minor nit should be "SCons", not "Scons"

Reply:
Whoops. I keep doing that, I will get to it tomorrow 😁